djstripe/fields.py

```diff
@@ -110,6 +110,10 @@ def stripe_to_db(self, data):
         """Convert the raw value to decimal representation."""
         val = data.get(self.name)
 
+        # If already a string, it's decimal in the API (eg. Prices).
+        if isinstance(val, str):
+            return decimal.Decimal(val)
```

Comment:
alternatively to adding a new field type, remove these new lines, then the field will use the normal behavior to store values as dollars?

(But I think adding a new field for decimal cents is a better option).